content/api/events.apib

```diff
@@ -126,6 +126,8 @@ Returns a list of message events that matched the filtered search.
         + Default: `1000`
     + delimiter (string, optional) ... Specifies the delimiter for query parameter lists.
         + Default: `,`
+    + event_ids (string, optional) - Comma delimited list of events, by ID, to search.
```

Comment:
`list of events` This might be a little confusing. How about just "Comma delimited list of event IDs to search"

Reply:
I wrote it that way originally and then debated which phrasing would better distinguish this from the 'events' parameter, but I'll go with your phrasing since I was conflicted anyway.